_data/handles/Moksha.json

```diff
@@ -1,5 +0,0 @@
-{
-  "handle": "Moksha",
```

Comment:
is this file part of updating styles?

Reply:
These are weird git cruft because we use user generated data as file names, and different operating systems handle capital letters differently. All things that should not be happening 😅